lib/buildFactory.js

```diff
@@ -168,7 +169,7 @@ class BuildFactory extends BaseFactory {
                     ]).then(([setup, teardown]) => {
                         modelConfig.steps = setup.concat(modelConfig.steps, teardown);
                         // Launcher is hardcoded to do some business in sd-setup-launcher
-                        modelConfig.steps.unshift({ name: 'sd-setup-launcher' });
+                        modelConfig.steps.unshift({ name: 'sd-setup-launcher', alwaysRun: true });
```

Comment:
I think it's ok to have it here. In case the launcher logic changes, this is still true.